templates/release-statefulset.yaml

```diff
@@ -120,6 +121,7 @@ spec:
               value: {{ include "rabbitmq.subchart" . }}
             - name: RABBITMQ_PORT
               value: {{ .Values.rabbitmq.service.ports.amqp | quote }}
+          resources: {{- include "common.resources.preset" (dict "type" "nano") | nindent 12 }}
```

Comment:
here and on other places for init-containers we should use constants from the values yaml for the waiting containers

Reply:
Nano size values used as constants.